extensions/web/08_text_encoding.js

```diff
@@ -4648,6 +4771,8 @@
 
   window.TextEncoder = TextEncoder;
   window.TextDecoder = TextDecoder;
+  window.TextEncoderStream = TextEncoderStream;
+  window.TextDecoderStream = TextDecoderStream;
```

Comment:
You will likely discover this during rebase, but instead of assigning right to window, assign to `window.__bootstrap.encoding` (like TextEncoder and TextDecoder on main). Then assign to window like seen here: https://github.com/denoland/deno/blob/main/runtime/js/99_main.js#L288-L289